sdks/python/apache_beam/ml/gcp/cloud_dlp.py

```diff
@@ -213,7 +213,7 @@ def setup(self):
       self.client = dlp_v2.DlpServiceClient()
     self.params = {
         'timeout': self.timeout,
-        "parent": self.client.project_path(self.project)
+        "parent": self.client.common_project_path(self.project)
```

Comment:
This is due to a backwards-incompatible change in google-cloud-dlp client library? Should we report it upstream?

Reply:
that seems to be the case, I couldn't find from which version the project_path starts missing though.

Reply:
cc: @minherz FYI

Reply:
thanx. this is not a change i made. probably comes from the lib refactoring.